composer.json

```diff
@@ -27,7 +26,12 @@
         "mockery/mockery": "^1.0",
         "orchestra/database": "^3.7",
         "orchestra/testbench": "^3.7",
-        "phpunit/phpunit": "^7.0"
+        "phpunit/phpunit": "^7.0",
+        "predis/predis": "^1.1"
+    },
+    "suggest": {
+        "ext-redis": "Required to use the redis cache and queue drivers (*).",
```

Comment:
Is the version correct here? Don't we need a specific version? I don't know if Horizon is compatible with every single version of phpredis.

Reply:
I used the composer.json for laravel/framework as an example. It requires extensions like this:
```
"ext-json": "*",
"ext-mbstring": "*",
"ext-openssl": "*",
```
and suggests two like this:
```
"ext-pcntl": "Required to use all features of the queue worker.",
"ext-posix": "Required to use all features of the queue worker.",
```
I'm not sure about possible compatibility issues.

Reply:
Let's perhaps leave the ` (*)` away from here in that case.

Reply:
Will do.